outputs/aws.go

```diff
@@ -135,6 +138,37 @@ func (c *Client) SendMessage(falcopayload types.FalcoPayload) {
 	c.PromStats.Outputs.With(map[string]string{"destination": "awssqs", "status": "ok"}).Inc()
 }
 
+// UploadS3 upload payload to S3
+func (c *Client) UploadS3(falcopayload types.FalcoPayload) {
+	f, _ := json.Marshal(falcopayload)
+
+	prefix := ""
+	if c.Config.AWS.S3.Prefix != "" {
```

Comment:
You need to declare the config https://github.com/falcosecurity/falcosidekick/blob/master/config.go